package.json

```diff
@@ -135,7 +133,6 @@
     "got": "^11.8.5",
     "graphql": "^15.8.0",
     "graphql-subscriptions": "^2.0.0",
-    "graphql-v14": "npm:graphql@14.7.0",
     "graphql-v16": "npm:graphql@16.6.0",
```

Comment:
_question:_ Not really related to your PR but for most libs I think we use the unqualified package name for the most recent version, for some reason for graphql we use `graphql` for 15 and graphql-v16 for 16. I wonder if we would want to switch that around. But not really important I guess.

Reply:
Yeah a little inconsistent. I want to update some libs versions on Monday. Will take a look too